releasenotes/notes/0.9/improvements-in-DAG-drawer-73283f18ba5ba937.yaml

```diff
@@ -0,0 +1,8 @@
+---
+upgrade:
+  - |
```

Comment:
Sorry this comment got lost because of a change made while I was reviewing:

Where was this introduced? I don't see any changes to the qiskit.visualization.dag_drawer() since 0.8. The only commit in the log that touches qiskit.visualization.dag_visualization was #2328 which just changed the error message if the dependencies weren't installed.

Reply:
Good catch! I think I confused it with PassManager drawer... #3005